tests/test_plugins_os_unix_config_default_parser.py

```diff
@@ -0,0 +1,51 @@
+from io import StringIO
+
+import pytest
+
+from dissect.target.plugins.os.unix.config import Default
+
+
+@pytest.mark.parametrize(
+    "parser_string, key, value",
+    [
+        ("hello world", "hello", "world"),
+        ("hello world\t# new info", "hello", "world"),
+    ],
+)
+def test_unknown_parser(parser_string: str, key: str, value: str):
```

Comment:
Add return type (None) to test functions? I believe this is the current practice?